wikispeedruns/prompts.py

```diff
@@ -249,12 +256,17 @@ def get_archive_prompts(prompt_type: PromptType, user_id: Optional[int]=None, of
     '''
     Get all prompts for archive, including currently active
     '''
-    if (prompt_type == "sprint"):
-        query = "SELECT prompt_id, start, end, rated, active_start, active_end, cmty_added_by, cmty_anonymous, cmty_submitted_time, username FROM sprint_prompts"
+    #if (prompt_type == "sprint"):
+    #    query = "SELECT prompt_id, start, end, rated, active_start, active_end, cmty_added_by, cmty_anonymous, cmty_submitted_time, username FROM sprint_prompts"
     # elif (prompt_type == "marathon")
```

Comment:
Oh wow, I guess we were thinking about this a while ago/ Also maybe just get rid of these comments.

Reply:
Removed